src/components/ComboBox/ComboBox.tsx

```diff
@@ -1,11 +1,13 @@
 import * as Popover from '@radix-ui/react-popover';
 import classnames from 'classnames';
 import { Command, CommandGroup, CommandInput, CommandItem, CommandList } from 'cmdk';
-import React from 'react';
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
 import { useOnClickOutside } from 'usehooks-ts';
 import * as iconComponents from '../../assets/formatted';
```

Comment:
this should be an explicit import for each icon otherwise code-splitting and tree-shaking doesn't work correctly